src/braket/circuits/ascii_circuit_diagram.py

```diff
@@ -216,21 +261,25 @@ def _ascii_diagram_column_set(
 
         first_line = "{:^{width}}|\n".format(col_title, width=len(lines[0]) - 1)
 
-        return first_line + "\n".join(lines)
+        return first_line + "\n".join(lines), global_phase
```

Comment:
Not a fan of this design pattern (function returning a tuple of two "unrelated" things). Can we make a separate helper function for this? I wouldn't be concerned about iterating through the moments twice